### PR TITLE
added ZendQueue\Adapter\Amqp

### PR DESCRIPTION
**Pull request by michaa76**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,14 +19,16 @@
             "url": "http://packages.zendframework.com/"
         }
     ],
+    "require": {
+        "php": ">=5.3.3",
+        "zendframework/zend-stdlib": "~2",
+        "videlalvaro/php-amqplib": "~2"
```
Comment (Maks3w) [inline composer.json]: Please place the dependency as a suggestion.

**Comment by Maks3w** [comment]
Please try to don't add unrelated changes in the same PR

**Comment by michaa76** [comment]
Maks3w could u please review the changes of mine?

File: library/ZendQueue/Adapter/Amqp.php [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+    *********************************************************************/
+
+    /**
+     * Returns the configuration options in this adapter.
+     *
+     * @return array
+    */
+    public function getOptions()
+    {
+        return parent::getOptions();
```
Comment (Maks3w) [inline library/ZendQueue/Adapter/Amqp.php]: Why overwrite this?

**Comment by Maks3w** [comment]
I see many CS issues and as I said there are changes could be done in a diffferent PR.

About the functionallity I don't have the knowdlege for to review.

File: library/ZendQueue/Adapter/Amqp.php (hunk 2 above)
Comment (michaa76) [inline library/ZendQueue/Adapter/Amqp.php]: will be removed in the next commit

**Comment by michaa76** [comment]
who can review the functionality and give u OK to get merged?

**Comment by weierophinney** [comment]
@michaa76 Honestly, we haven't been supporting this component for years, due in large part to large architectural issues. I recommend forking.

**Pull request closed without merging**